### PR TITLE
feat: update ref

### PR DESCRIPTION
**Pull request by laurentsimon**
Need https://github.com/slsa-framework/slsa-github-generator/pull/1087 and https://github.com/slsa-framework/slsa-github-generator/pull/1107 to be merged first.

### CODE REVIEW

**Comment by laurentsimon** [comment]
Note: the reported provenance will show a workingDir as `/path/to/github-workspace/project-name/__PROJECT_CHECKOUT_DIR__ /` whereas it was `/path/to/github-workspace/project-name/project-name/` before

File: .github/workflows/builder_node_slsa3.yml [hunk 1]
```diff
@@ -229,33 +229,36 @@ jobs:
...
+        working-directory: __PROJECT_CHECKOUT_DIR__
         env:
           UNTRUSTED_CI_ARGUMENTS: "${{ inputs.ci-arguments }}"
           UNTRUSTED_DIR: ${{ inputs.directory }}"
         run: |
           set -euo pipefail
 
           # npm ci <args>
-          ./"$BUILDER_BINARY" ci \
+          ../"$BUILDER_BINARY" ci \
```
Comment (ianlewis) [inline .github/workflows/builder_node_slsa3.yml:240]: nit: this is ok, but assumes that the `__PROJECT_CHECKOUT_DIR__` is one directory deep. consider using `$GITHUB_WORKSPACE`

**Review by ianlewis — APPROVED** [review]

File: .github/workflows/builder_node_slsa3.yml (hunk 1 above)
Comment (laurentsimon) [reply .github/workflows/builder_node_slsa3.yml:240]: great idea. I made the changes.

Comment (ianlewis) [reply .github/workflows/builder_node_slsa3.yml:240]: It seems you got everywhere else except this line ;)

**Review by ianlewis — APPROVED** [review]

**Pull request merged**